third-party/llvm/Makefile.include-llvm

```diff
@@ -12,9 +12,17 @@ ifneq ("$(wildcard $(LLVM_POLLY_FILE))","")
 endif
 
 ifndef LLVM_LLVM_LIBS
+
+ifeq ($(LLVM_MAJOR_VERSION),10)
+  # frontendopenmp seems to be required to link clang in llvm 10
+  # but not in earlier or later versions
+  export LLVM_LLVM_LIBS=$(shell test -x $(LLVM_CONFIG) && $(LLVM_CONFIG) --ldflags --system-libs --libs bitreader bitwriter frontendopenmp ipo instrumentation option objcarcopts profiledata all-targets coverage coroutines lto)
```

Comment:
Can we simplify this and the below CLANG_LIBS by just appending `frontendopenmp` if we're using llvm 10?

i.e. 
```
export LLVM_LLVM_LIBS=$(shell test -x $(LLVM_CONFIG) && $(LLVM_CONFIG) --ldflags --system-libs --libs bitreader bitwriter ipo instrumentation option objcarcopts profiledata all-targets coverage coroutines lto)
ifeq ($(LLVM_MAJOR_VERSION),10)
  export LLVM_LLVM_LIBS=$(LLVM_LLVM_LIBS) frontendopenmp
endif

```

Reply:
I wrote it this way because this way llvm-config gives us the link args in a reasonable order (i.e. I'm not sure what the dependencies of frontendopenmp are - so I'm not sure we can put it at the end like that). Anyway in your example we would be adding -lLLVMFrontendOpenMP since frontendopenmp is the name of a "component" not a library.

Reply:
Ah, I see -- nevermind then